client/connect-account-page/strings.js

```diff
@@ -3,42 +3,60 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { __experimentalCreateInterpolateElement as createInterpolateElement } from 'wordpress-element';
 
 export default {
+	button: __( 'Finish Setup', 'woocommerce-payments' ),
```

Comment:
I am not a fan of this approach of separating out the strings into a separate file. It obfuscates things by making it more difficult to read the JSX source and understand what the UI will look like. It feels like a step backwards as far as translation techniques are concerned (it reminds me of looking up resource strings in RC files via defines for Windows apps).

I know this isn't a technique introduced in this PR though, and if this is the approach favored by `woocommerce-payments`, then I am okay with this as far as this PR is concerned.

Reply:
I totally 💯 agree, but it is a existing convention in WC Pay so I would prefer to keep it in this PR at least.

> It obfuscates things by making it more difficult to read the JSX source and understand what the UI will look like.

FWIW I found working on this PR tedious at times because of these exact reasons.